python/ambassador/ir/irerrorresponse.py

```diff
@@ -232,6 +278,20 @@ def _generate_mappers(self) -> Optional[List[Dict[str, Any]]]:
 
                 body_format_override["content_type"] = ir_content_type
 
+            # search the body for command tokens
+            # TODO: remove this code when envoy supports escaping "%"
+            token_finder = re.compile(ENVOY_FMT_TOKEN_REGEX)
+            matches = token_finder.findall(format_body)
+
+            bad_token = False
+            for i in matches:
+                if not i[0] in ALLOWED_ENVOY_FMT_TOKENS:
```

Comment:
This is probably worth a comment, just in case the reader isn't familiar with what a match looks like. (NGL, I don't remember what the tuple contains.)

Reply:
Will add a comment!